pkg/apis/pipeline/v1beta1/task_types.go

```diff
@@ -125,8 +125,15 @@ type Step struct {
 	Script string `json:"script,omitempty"`
 }
 
-// A sidecar has the same data structure as a Step, consisting of a Container, and optional Script.
-type Sidecar = Step
+// Sidecar has nearly the same data structure as Step, consisting of a Container and an optional Script, but does not have the ability to timeout.
```

Comment:
minor nit: suggest matching comment style with others in the file and wrapping the lines around 80-100 characters.

similarly for other comments in this pr.

Reply:
Updated in new commit!